app/helpers/guide_helper.rb

```diff
@@ -36,10 +36,12 @@ def guide_community_options_for_select
 
   def topic_section_options_for_select
     Topic.includes(:topic_sections).map do |topic|
+      relative_path = topic.path.gsub("/service-manual", "")
```

Comment:
Probably unlikely, but I think if we had a meta topic e.g. 'Service Manual Usage' that lived at '/service-manual/service-manual-usage` we might break things with this. Regex with start of string anchor?